.golangci.yml

```diff
@@ -0,0 +1,12 @@
+linters-settings:
```

Comment:
what is this?

Reply:
It‘s a configure for depguard. reference for https://github.com/golangci/golangci-lint/issues/3877#issuecomment-1573682848
The default rule of depguard is to allow only std lib in all files.